### PR TITLE
Gita

### PR DESCRIPTION
**Pull request by fitristachan**

### CODE REVIEW

**Comment by fitristachan** [comment]
plis bantu merge, takut gua. makasii sebelumnya.

File: app/Config/Routes.php [hunk 1]
```diff
@@ -29,11 +29,13 @@
...
-$routes->get("/spk/", "SPK::index");
-$routes->post('/spk/submit', 'SPK::submit_alternative');
-$routes->get('/nextpage', 'SPK::index');
-$routes->get('/spk/result/(:num)', 'SPK::result/$1');
+
+$routes->get('/', 'Auth::index');
+ 
+$routes->get('/auth/(:any)', 'Auth::$1');
+$routes->post('/auth/(:any)', 'Auth::$1');
+
```
Comment (rianfrhn) [inline app/Config/Routes.php:37]: tambahin
$routes->get("/spk/", "SPK::index");
$routes->post('/spk/submit', 'SPK::submit_alternative');
$routes->get('/nextpage', 'SPK::index');
$routes->get('/spk/result/(:num)', 'SPK::result/$1');

File: app/Config/Database.php
Comment (rianfrhn) [inline app/Config/Database.php:1]: keknya jangan setup databasenya disini. Pake .env aja

**Review by rianfrhn — CHANGES_REQUESTED** [review]
route tambahin:
$routes->get("/spk/", "SPK::index");
$routes->post('/spk/submit', 'SPK::submit_alternative');
$routes->get('/nextpage', 'SPK::index');
$routes->get('/spk/result/(:num)', 'SPK::result/$1');

Database jg keknya pake .env aja biar ga ribet setting" ulang tiap kali merge

File: app/Config/Routes.php [hunk 2]
```diff
@@ -48,6 +50,16 @@
  * You will have access to the $routes object within that file without
  * needing to reload it.
  */
+//home
+$routes->get('/home/(:any)', 'Home::$1');
+$routes->post('/home/(:any)', 'Home::$1');
+//,['filter' => 'authGuard']
+
+$routes->get("/spk/", "SPK::index");
```
Comment (fitristachan) [inline app/Config/Routes.php:58]: @rianfrhn iniii

Comment (rianfrhn) [reply app/Config/Routes.php:58]: ow iya

**Review by rianfrhn — APPROVED** [review]

**Pull request merged**